src/components/timeline/RecentTimeline.js

```diff
@@ -1,25 +1,63 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import axios from 'axios';
-import { useQuery } from 'react-query';
 import { Timeline, TimelineItem } from 'vertical-timeline-component-for-react';
-import Moment from 'react-moment';
+import { DateTime } from 'luxon';
+import { nanoid } from 'nanoid';
+import { useTimeline } from '../../state/query_hooks/useTimeline';
+import './RecentTimeline.css';
 
 export function RecentTimeline() {
-  useQuery('timeline', async () => {
-    const { data } = await axios.get(
-      `${process.env.REACT_APP_BACKENDURL}/incidents/getincidents`
-    );
+  const timelineQuery = useTimeline();
 
-    return data;
-  });
+  const urlDomain = url => {
```

Comment:
Is this a feature that we want to use elsewhere? If so we should extract the logic so that it can be used elsewhere. Let's discuss.